src/commands/runInTerminalCommand.ts

```diff
@@ -7,13 +9,17 @@ interface RunInTerminalArgs {
 	cwd?: string;
 	reveal?: boolean;
 	waitForExit?: boolean;
-	reuse?: boolean;
+	reuse?: string;
 }
 
 let onDidCloseTerminalDisposable: Disposable;
 
 export async function runInTerminalCommand(arg: RunInTerminalArgs | string): Promise<unknown> {
 	return new Promise((resolve, reject) => {
+		if (!arg) {
```

Comment:
I had forgotten that I'd added this check - previously it was actually possible to create a terminal without sending text, although I don't know whether that was intentional as it wasn't documented.